docs/source/quickstart/02_prepare_env.md

```diff
@@ -22,13 +22,19 @@ There are a few needed dependencies to install before starting consuming the fra
 
 ## On the hypervisor
 
-On the hypervisor, please ensure you have:
+On the hypervisor host, please ensure you have:
 
 - a non-root user, with passwordless SSH access (use SSH keys)
 - `sudo` configuration allowing that non-root user to run any random command, with or without password
 - at least 400G of free space in /home
 - an up-to-date CentOS Stream 9 or RHEL-9.2 system
 
+Besides the `qemu-kvm` and the `libvirt` packages, you will also need to have `tar` and `rsync`:
```

Comment:
qemu-kvm and libvirt should be installed via the libvirt_manager - there shouldn't be manual install at this point.

same goes for tar and rsync - imho we should inject those two in the ci_setup role - see https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/ci_setup/vars content for instance.

Reply:
I didn't automate its installation because sometimes setting the hypervisor requires extra manual steps to do it right.

I usually run `virt-host-validate` to check out the status after installing libvirt and qemu/kvm to see what's missing.

In my server I had to enable IOMMU and reboot before using ci-framework:
```
$ sudo grubby --update-kernel ALL --args intel_iommu=on
$ sudo grubby --update-kernel ALL --args iommu=pt
```

But I'm ok adding it to the automation.

Reply:
we can also get "unzip" in the ci_setup package list. I happen to be redeploying a hypervisor from 0 right now and I'm seeing those missing packages as well.
Also, the "sudo" injected by the rhol_crc role isn't covering the whole setup anymore - guess we'll go with a full NOPASSWD:ALL - I'll set this up in the ci_setup...

Reply:
Now that I think about it, one of the steps talks about running `sudo virsh net-destroy default`, so if we are installing the packages, shouldn't we also run that command with ansible?

Reply:
that `net-destroy default` is temporary. I have to get some time to check if we can re-use that default network as-is, or if we can change "our" custom net range to not overlap.